build.sbt

```diff
@@ -55,6 +60,8 @@ val timeBenchmark =
     .enablePlugins(JmhPlugin)
     .settings(commonSettings: _*)
     .settings(
+      // FIXME: does not compile with dotty
+      crossScalaVersions := List(scalaVersion.value),
```

Comment:
We can just update the dotty version in the build.sbt to be a nightly build with the fix in.